oracle/oracle.go

```diff
@@ -12,25 +12,31 @@ import (
 	"github.com/oracle/oci-go-sdk/v65/common"
 	"github.com/oracle/oci-go-sdk/v65/containerengine"
 	"github.com/oracle/oci-go-sdk/v65/core"
-	"github.com/sirupsen/logrus"
 )
 
 const (
-	v1MetadataAPIEndpoint         = "http://169.254.169.254/opc/v1/instance/"
-	v2MetadataAPIEndpoint         = "http://169.254.169.254/opc/v2/instance/"
-	metadataInstanceIDkey         = "id"
-	metadataRegionKey             = "region"
-	metadataAvailabilityDomainKey = "availabilityDomain"
+	v1MetadataAPIEndpoint = "http://169.254.169.254/opc/v1/instance/"
+	v2MetadataAPIEndpoint = "http://169.254.169.254/opc/v2/instance/"
+	metadataInstanceIDkey = "id"
+	// MetadataRegionKey key name for region in metadata JSON
+	// returned by IMDS service
+	MetadataRegionKey = "canonicalRegionName"
+	// MetadataAvailabilityDomainKey key name for availability domain
+	// in metadata JSON returned by IMDS service
+	MetadataAvailabilityDomainKey = "availabilityDomain"
```

Comment:
Let's stick with this for now. May be during display we can think of getting rid of the tenancy name. Can you file a ticket to revisit it later. We'll have to see in our commands output and make sure it makes sense.

Reply:
Filed https://portworx.atlassian.net/browse/PWX-26183 for this.